docker-images/kafka/Dockerfile

```diff
@@ -44,6 +34,10 @@ RUN curl -O https://archive.apache.org/dist/kafka/${KAFKA_VERSION}/kafka_${SCALA
 
 COPY ./tmp/kafka-agent-${strimzi_version}.jar ${KAFKA_HOME}/libs/
 
+COPY thirdparty-libs/ ${KAFKA_HOME}/libs/
+
+COPY --from=THRIDPARTY_LIB_BUILD ${KAFKA_HOME}/libs/ ${KAFKA_HOME}/libs/
```

Comment:
This seems to be a left over from the multi stage build which should be removed.